code/extraction.py

```diff
@@ -119,51 +117,51 @@ def make_output_directory(output_dir: Path, experiment_id: str) -> str:
     return output_dir
 
 
-def write_output_metadata(
+def write_data_process(
     metadata: dict,
-    process_json_dir: str,
-    process_name: str,
     input_fp: Union[str, Path],
     output_fp: Union[str, Path],
-    start_date_time: dt,
+    unique_id: str,
+    start_time: dt,
+    end_time: dt,
 ) -> None:
-    """Writes output metadata to plane processing.json
+    """Writes output metadata to plane data_process.json
 
     Parameters
     ----------
     metadata: dict
-        parameters from suite2p cellpose segmentation
-    input_fp: str
-        path to data input
+        parameters from suite2p motion correction
+    raw_movie: str
```

Comment:
The argument is named `input_fp`. to be pydantic, while the input is indeed a movie, it should be motion corrected (and decrosstalked) not "raw"

Reply:
fixed